iam/iam-for-deploy-bot/main.tf

```diff
@@ -352,15 +114,47 @@ resource "aws_iam_group" "deploy-bot-deploy-access" {
   name = "deploy-bot-deploy-access"
 }
 
+resource "aws_iam_policy" "deploy-bot-extra-access" {
+  name        = "DeployBot_ExtraAccess"
+  path        = "/"
+  description = "Allow extra access for deploy-bot"
```

Comment:
NIT: can we document here **why** we need this additional access? Maybe we could also find a better name than "extra" to clarify what this is for.